web/src/app/modules/actions/modules/common-controls/components/common-controls.component.ts

```diff
@@ -38,6 +37,9 @@ export class CommonControls {
         if (this.isSaveEnabled) {
             await this.validator.validateCurrent();
             if (this.isSaveEnabled && this.validator.isSavingEnabled()) {
+                if (this.isModel()) {
+                    await this.dataService.saveModelImage(this.navigator.currentElement);
+                }
                 this.dataService.commit(this.translate.instant('save'));

```

Comment:
It seems strange that first the model image is saved and then the model itself. Wouldn't the other way round be better? What when the saving gives an error (e.g. because of a validator)